doc/howto/projects_create.md

```diff
@@ -19,6 +19,11 @@ To create a project called `my-restricted-project` that blocks access to securit
 
     lxc project create my-restricted-project --config restricted=true --config restricted.backups=allow
 
+```{tip}
+When you create a project without specifying configuration options, {config:option}`project-features:features.profiles` is set to `true`, which means that profiles are isolated in the project.
+Consequently, the new project does not have access to the `default` profile of the `default` project and therefore misses essential configuration (like the root disk).
```

Comment:
I think we should expand this to say something like:

> ..the new project does not have access to the `default` profile of the `default` project and therefore misses required configuration for creating instances (like the root disk). This can be rectified by adding a root disk device to the project's default profile using `lxc profile device add`.